state/stateenvirons/policy.go

```diff
@@ -41,37 +43,25 @@ type deployChecker interface {
 }
 
 type storageServiceGetter func(modelUUID coremodel.UUID, registry storage.ProviderRegistry) state.StoragePoolGetter
+type modelServiceGetter func(modelUUID coremodel.UUID) ModelConfigService
```

Comment:
Let's call this `modelConfigServiceGetter`, "model service" is a different thing entirely. Or better yet, we can just use a function type, we may not need this type alias.